actors/verifreg/src/events.rs

```diff
@@ -0,0 +1,105 @@
+use crate::{Allocation, AllocationID, Claim, ClaimID, DataCap};
+use fil_actors_runtime::runtime::Runtime;
+use fil_actors_runtime::{ActorError, EventBuilder};
+use fvm_shared::ActorID;
+
+// A namespace for helpers that build and emit verified registry events.
+// REVIEW: would this be better as a simple module with pub fns, called as emit::verifier_balance()?
```

Comment:
Generally, yes. Either that or a wrapper on the Runtime. E.g., `Emit<R: Runtime(&R)`. But, honestly, I'd prefer the module.